examples/specs/facet_custom_header.vl.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Correct me if I'm wrong -- isn't this example an exact duplicate of `examples/specs/facet_custom.vl.json`?

Reply:
They are equivalent, but one is a shorthand.